ColorCodeMgr.cpp

```diff
@@ -0,0 +1,29 @@
+#include "ColorCoderMgr.h"
+
+namespace TelCoColorCoder
+{
+    ColorPair CColerCoderMgr::GetColorFromPairNumber(int pairNumber)
+    {
+        int zeroBasedPairNumber = pairNumber - 1;
+        MajorColor majorColor =
+            (MajorColor)(zeroBasedPairNumber / ColorPair::numberOfMinorColors);
+        MinorColor minorColor =
+            (MinorColor)(zeroBasedPairNumber % ColorPair::numberOfMinorColors);
+        return ColorPair(majorColor, minorColor);
+    }
+
+    int CColerCoderMgr::GetPairNumberFromColor(MajorColor major, MinorColor minor)
+    {
+        return major * ColorPair::numberOfMinorColors + minor + 1;
+    }
+
+    std::string CColerCoderMgr::GetColorCodeManual()
```

Comment:
The "color code manual" function is different from the other two - e.g., it can change when the language changes. The name of the class is one clue that we're putting unrelated stuff in the same class: Names like handler, manager are generic and indicate that the file will collect unrelated functionalities.